.changes/unreleased/FEATURES-20240912-140136.yaml

```
@@ -0,0 +1,5 @@
+kind: FEATURES
+body: '* resource/archive_file: Remove `deprecated` status'
```

```suggestion
body: 'resource/archive_file: Remove `deprecated` status'
```
Changie will add the bullet point when creating the Changelog.